tests/model/selection.js

```diff
@@ -1237,4 +1237,56 @@ describe( 'Selection', () => {
 			} );
 		} );
 	} );
+
+	describe( 'isEntireContentSelected()', () => {
```

Comment:
Missing tests:

* empty content,
* empty selection at a boundary of non-empty content